coreforge/mpact_builder/infinite_medium.py

```diff
@@ -0,0 +1,128 @@
+from typing import TypedDict, Optional
+from dataclasses import dataclass
+from math import inf
+
+import mpactpy
+
+from coreforge.mpact_builder.mpact_builder import register_builder, build_material
+from coreforge.mpact_builder.builder_specs import BuilderSpecs
+from coreforge.mpact_builder.material_specs import MaterialSpecs
+from coreforge import geometry_elements
+
+@register_builder(geometry_elements.InfiniteMedium)
+class InfiniteMedium:
+    """ An MPACT geometry builder class for InfiniteMedium
+
+    Parameters
+    ----------
+    specs: Optional[Specs]
+        Specifications for building the MPACT representation of this element
+
+    Attributes
+    ----------
+    specs: Optional[Specs]
+        Specifications for building the MPACT representation of this element
+    """
+
+    @dataclass
+    class Specs(BuilderSpecs):
+        """ Building specifications for InfiniteMedium
+
+        Attributes
+        ----------
+        target_cell_thicknesses : Thicknesses
+        The target side length of the cells (cm).  Defaults to infinity for dimensions
+        where no targets provided. (keys: "X", "Y", "Z")
+        thicknesses : Thicknesses
+            The thicknesses of each material region division in each axis-direction
+            (keys: "X", "Y", "Z")
+        height : float
+            The height to build the extruded PinCell in the axial direction (cm).
+            Default value is 1.0
+        divide_into_quadrants : bool
+            An optional setting to divide the pincell into 4 separate MPACT Module quadrants.
+            This will represent the pincell with 4 MPACT Modules rather than just one.
+            Default value is False
+        material_specs : MaterialSpecs
+            Specifications for how materials should be treated in MPACT
+        """
+
+        class Thicknesses(TypedDict):
+            """ Class specifying the keys for target cell thickness
+            """
+            X: float
+            Y: float
+
+        thicknesses:             Thicknesses
+        target_cell_thicknesses: Optional[Thicknesses] = None
+        height:                  float = 1.0
+        divide_into_quadrants:   bool = False
+        material_specs:          Optional[MaterialSpecs] = None
+
+        def __post_init__(self):
+            if not self.target_cell_thicknesses:
+                self.target_cell_thicknesses = {"X": inf, "Y": inf, "Z": inf}
+
+            if not self.material_specs:
+                self.material_specs = MaterialSpecs()
+
+            assert all(thickness > 0. for thickness in self.target_cell_thicknesses.values()), \
+                f"target_cell_thicknesses = {self.target_cell_thicknesses}"
+
+            assert self.height > 0., f"height = {self.height}"
+
+            assert all(thickness > 0. for thickness in self.thicknesses.values()), \
+                f"target_cell_thicknesses = {self.thicknesses}"
```

Comment:
```suggestion
                f"thicknesses = {self.thicknesses}"
```
Maybe a copy-paste error from line 70?

Reply:
Fixed